playbooks/roles/mattermost/defaults/main.yml

```diff
@@ -4,6 +4,9 @@ MATTERMOST_HOSTNAME: "{{ inventory_hostname }}"
 MATTERMOST_OPS_EMAIL: ops@exmaple.com
 
 MATTERMOST_VERSION: 5.8.0
+
+# UNLINK is the autolink plugin which converts JIRA tickets to clickable URL
+# https://github.com/mattermost/mattermost-plugin-autolink/releases/
 MATTERMOST_UNLINK_VERSION: 0.3.0
```

Comment:
~~Can we update the default to `1.1.0` also?~~

Actually, can we remove this default and move the comment to the secrets repo? I'm not a big fan of defaults for vars that don't really have any meaningful universal fall-back value.

In the case of versions, that's either "latest" or, for some software packages, a well-recognized standard value, such as "2.7" or "3.6". </soapbox> :)

Reply:
Done.